cmake/modules/HCT.cmake

```diff
@@ -49,6 +58,12 @@ function(add_hlsl_hctgen mode)
                        ${hctdb}
                        ${hctdb_helper})
 
+  get_filename_component(output_extension ${full_output} LAST_EXT)
+
+  if (CLANG_FORMAT_EXE AND output_extension MATCHES "\.h|\.cpp|\.inl")
```

Comment:
We may have agreed on this already, but do we want to run clang-format on our hlsl test files? Does it even work on those files?

Reply:
HLSL source files should clang-format just fine. This line isn’t about that though. This code is only used to run clang-format on the source files that contain bits of generated code that come out of the hctgen scripts.